src/highlight.js

```diff
@@ -274,18 +274,6 @@ https://highlightjs.org/
     return [mode];
   }
 
-  function restoreLanguageApi(obj) {
-    if(API_REPLACES && !obj.langApiRestored) {
```

Comment:
Do we still need `API_REPLACES` variable declared above?

Reply:
The existing build system hooks on it I think - and this branch doesn't touch the build system.  I'll get it removed after the two branches are merged though.